src/tokens/Assets.test.tsx

```diff
@@ -336,6 +336,21 @@ describe('AssetsScreen', () => {
     expect(navigate).toHaveBeenCalledWith(Screens.DappShortcutsRewards)
   })
 
+  it('clicking Import Token opens a screen', () => {
+    jest.mocked(getFeatureGate).mockReturnValue(true)
+    const store = createMockStore(storeWithPositionsAndClaimableRewards)
+
+    const component = (
+      <Provider store={store}>
+        <MockedNavigator component={AssetsScreen} />
+      </Provider>
+    )
+    const { getByTestId } = render(component)
+    fireEvent.press(getByTestId('Assets/ImportTokenButton'))
```

Comment:
Nit: we aim to use `getByTestId` as little as possible unless there is a strong reason (e.g., testing which text is displayed).

In this case, if we use the `getByText` selector, the test behavior would be closer to the actual user experience.